docs/arcaflow/getting-started.md

```diff
@@ -0,0 +1,249 @@
+# Arcaflow Getting Started Guide
+
+## Introduction
+
+[Arcaflow](https://arcalot.io/arcaflow/) is a modular system that enables engineers to easily build complex parallelized workflows for test automation without requiring any pre-installation or deployment of prerequisite software stacks. It uses an **engine** and **plugin** architecture, together with which **workflows** are constructed. Workflows are composable and highly portable, capable of orchestrating complex interrelated actions between plugins. This effectively allows for engineering workflow expertise to be packaged up, version controlled, and shared for repeatability across environments and platforms.
+ 
+The engine component is intended to run from your laptop, jump host, or wherever you have network connectivity to the System Under Test (SUT). No installation is required of any Arcaflow components, either on your system or the SUT. The engine processes the workflow definition and passes the required data to the plugins.
```

Comment:
s/System Under Test (SUT)/target environment/?

Reply:
Perhaps you'd want to say that "the engine invokes the plugins in the target environment according to the workflow definition and passes the required data to the plugins as directed by the workflow" ... or something like that.